OpenDreamRuntime/Objects/DreamObject.cs

```diff
@@ -30,14 +30,18 @@ public class DreamObject {
             return new InitDreamObjectState(thread, this, usr, arguments);
         }
 
-        public static DreamObject GetFromReferenceID(IDreamManager manager, int refID) {
+        public static DreamObject? GetFromReferenceID(IDreamManager manager, int refID) {
             foreach (KeyValuePair<DreamObject, int> referenceIDPair in manager.ReferenceIDs) {
                 if (referenceIDPair.Value == refID) return referenceIDPair.Key;
             }
 
             return null;
         }
 
+
+        /// <summary>
+        /// Do NOT use this directly, instead use IDreamManager.CreateRef()
+        /// </summary>
```

Comment:
This should be moved to inside `DreamManager.CreateRef()` to prevent misuses & confusion. It only operates on a dictionary located in `DreamManager` anyway, there's no longer any reason for it to be here.